users/ollyhayes/ollyhayes.c

```diff
@@ -0,0 +1,203 @@
+/* Copyright 2022 Olly Hayes (@ollyhayes)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "ollyhayes.h"
+
+#include <stdio.h>
+#ifdef VIRTSER_ENABLE
+#   include "virtser.h"
+#endif
+
+static void sendUtf(const char *str) {
```

Comment:
Why not just use unicode?  You're essentially duplicating the code here, anyways. 

https://docs.qmk.fm/#/feature_unicode

Reply:
Specifically, you can use `send_unicode_string(str)` for this instead.  You'd want to enable unicode, and you could use `UNICODE_COMMON = yes` in your rules, and just use the core unicode framework.  You'd just need to set the mode.

Reply:
Thanks! I switched to using the unicode feature and added the commit to the review.